index.bs

```diff
@@ -28,6 +28,12 @@ spec: media-source; urlPrefix: https://www.w3.org/TR/media-source/
     type: method
         for: MediaSource; text: isTypeSupported(); url: #dom-mediasource-istypesupported
 
+spec: media-source-codec-switching; urlPrefix: https://rawgit.com/wicg/media-source/codec-switching/index.html
```

Comment:
Needs updating now that [changeType](https://www.w3.org/TR/media-source-2/#dom-sourcebuffer-changetype) is in MSE.

Reply:
Hmm... citing WICG is not great. 

If we want to add something from the WICG then it should be moved to a W3C working group.

Reply:
I updated the reference to media-source-2. I'm no Bikeshed expert, but I found it was able to autolink `SourceBuffer` and `changeType()` without entries in the anchors section.